config/install/views.view.roles_view.yml

```diff
@@ -0,0 +1,451 @@
+            html: false
+          element_type: ''
+          element_class: ''
+          element_label_type: strong
+          element_label_class: ''
+          element_label_colon: false
+          element_wrapper_type: ''
+          element_wrapper_class: ''
+          element_default_classes: true
+          empty: ''
+          hide_empty: false
+          empty_zero: false
+          hide_alter_empty: true
+          click_sort_column: value
+          type: timestamp_ago
+          settings:
+            future_format: '@interval hence'
+            past_format: '@interval ago'
+            granularity: 2
+          group_column: value
+          group_columns: {  }
+          group_rows: true
+          delta_limit: 0
+          delta_offset: 0
+          delta_reversed: false
+          delta_first_last: false
+          multi_type: separator
+          separator: ', '
+          field_api_classes: false
+        field_name:
+          id: field_name
+          table: user__field_name
+          field: field_name
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: field
+          label: ''
+          exclude: false
+          alter:
+            alter_text: true
+            text: "<div class=\"field field--name-field-name field--type-string field--label-above\">\r\n    <div class=\"field__label\">First name</div><div class=\"field__item\">{{ field_name }}</div>\r\n</div>"
+            make_link: false
+            path: ''
+            absolute: false
+            external: false
+            replace_spaces: false
+            path_case: none
+            trim_whitespace: false
+            alt: ''
+            rel: ''
+            link_class: ''
+            prefix: ''
+            suffix: ''
+            target: ''
+            nl2br: false
+            max_length: 0
+            word_boundary: true
+            ellipsis: true
+            more_link: false
+            more_link_text: ''
+            more_link_path: ''
+            strip_tags: false
+            trim: false
+            preserve_tags: ''
+            html: false
+          element_type: ''
+          element_class: ''
+          element_label_type: ''
+          element_label_class: ''
+          element_label_colon: false
+          element_wrapper_type: ''
+          element_wrapper_class: ''
+          element_default_classes: true
+          empty: ''
+          hide_empty: true
+          empty_zero: false
+          hide_alter_empty: true
+          click_sort_column: value
+          type: string
+          settings:
+            link_to_entity: false
+          group_column: value
+          group_columns: {  }
+          group_rows: true
+          delta_limit: 0
+          delta_offset: 0
+          delta_reversed: false
+          delta_first_last: false
+          multi_type: separator
+          separator: ', '
+          field_api_classes: false
+        field_last_name:
```

Comment:
hey @yeniatencio , does all sites has `field_first_last` and `field_name` field?

Reply:
as this is a new feature, please also add a Behat.

Reply:
Hey @vincent-gao,  content-reference doesn't have first name, last name and department fields. Content-vic does. 
![Screenshot 2023-05-04 at 9 43 49 am](https://user-images.githubusercontent.com/47239456/236074543-3904999a-a8d0-4f93-9730-077ec3ba7438.png)
![Screenshot 2023-05-04 at 9 42 43 am](https://user-images.githubusercontent.com/47239456/236074549-16523b11-b9a6-43a1-a90a-76c1ce838b8c.png)
 This is how they look.

Reply:
Hey @yeniatencio , they don't show up because the handlers are broken. 
![CleanShot 2023-05-04 at 16 25 50](https://user-images.githubusercontent.com/8788145/236126644-40163145-97bd-4ac6-bc94-9775707735d1.jpg)
adding field_last_name and field_name and an unknown field to the view that locates [config/install/views.view.roles_view.yml](https://github.com/dpc-sdp/tide_core/pull/398/files/df13cbe4fbbf6c069dcea7dde8c2053b8b7e47a4#diff-388c1ba55721a7f6601e3799ae9154f197d355e9b49d1573bbc361cf20bfebce) is not the right thing to do, because tide_core doesn't define `field_last_name`, `field_name` and the unknown field.

Reply:
@yeniatencio 
If there are fields in the project that differ from this module, you can override the module's configuration from the project.